test/resolvers_test.go

```diff
@@ -699,7 +752,7 @@ spec:
 		},
 		Type: corev1.SecretTypeOpaque,
 		Data: map[string][]byte{
-			scmTokenSecretKey: []byte(base64.StdEncoding.Strict().EncodeToString([]byte(token))),
+			scmTokenSecretKey: []byte(token),
```

Comment:
I'm not sure why this was base64 encoded on the application side. When I made the gitea repo private, API auth also broke until I removed this base64 encoding. I think it was mistakenly though to be required, however the base64 encoding looks to be happening server-side after the secret is created.